contrib/packs/actions/download.yaml

```diff
@@ -5,23 +5,16 @@
   enabled: true
   entry_point: "pack_mgmt/download.py"
   parameters:
-    packs:
-      type: "array"
-      items:
-        type: "string"
-    repo_url:
+    pack:
```

Comment:
We have a description in `install.meta.yaml`, you're not really supposed to launch `download` on its own. :) Still, fair point.